WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -5012,6 +5012,7 @@
 					"-l\"sqlite3.0\"",
 					"-l\"z\"",
 				);
+				OTHER_SWIFT_FLAGS = "-D DEBUG";
```

Comment:
I am not too familiar with this so feel free to ignore, but wherever I look the flags are always set without using a space between `-D` and `{FLAG}`. I've tested the code and it's working, so maybe it doesn't matter, but I wanted to bring it up anyway. As an example:

http://stackoverflow.com/questions/24111854/in-absence-of-preprocessor-macros-is-there-a-way-to-define-practical-scheme-spe